pipelines/deploy-fbc-operator/0.1/deploy-fbc-operator.yaml

```diff
@@ -157,9 +157,19 @@ spec:
                 exit 0
               fi
 
+              # Check if the bundle image is pullable before anything else
+              if skopeo inspect --no-tags --config "docker://${BUNDLE_IMAGE}" > /dev/null 2>&1; then
+                echo "Bundle image $BUNDLE_IMAGE is pullable. Using as-is, without applying mirror substitution."
+                echo -n "$BUNDLE_IMAGE" > "$(results.unreleasedBundle.path)"
+                exit 0
+              fi
+              echo "Could not inspect original pullspec $BUNDLE_IMAGE. Checking if there's a mirror present"
+              
+              # Fetch and process mirror set
               mirror_set_url="${SOURCE_GIT_URL}/raw/${SOURCE_GIT_REVISION}/.tekton/images-mirror-set.yaml"
               
-              if mirror_set_yaml=$(curl -sfL "${mirror_set_url}"); then
+              # Fetch mirror map from the URL
+              if mirror_set_yaml=$(curl -kfL "${mirror_set_url}"); then
```

Comment:
`curl -kfL` assumes that the git repo is public, right? What if the git repo requires a secret?

Reply:
yes, it assumes the repo is public. Could there be a case where product teams use a repo that is private?

Reply:
I encountered a number of private repos when generating PRs/MRs for the pruning graph check.

Reply:
If you want to address this in a follow-up PR to unblock this PR, that is okay with me.

Reply:
I'll address this in the follow-up PR, thank you, Nate!